src/main/resources/google_checks.xml

```diff
@@ -81,16 +81,16 @@
       <property name="id" value="LeftCurlyEol"/>
       <property name="tokens"
                value="ANNOTATION_DEF, CLASS_DEF, CTOR_DEF, ENUM_CONSTANT_DEF, ENUM_DEF,
-                    INTERFACE_DEF, LAMBDA, METHOD_DEF, OBJBLOCK, LITERAL_SWITCH,
-                    STATIC_INIT, RECORD_DEF, COMPACT_CTOR_DEF, LITERAL_CATCH, LITERAL_TRY,
-                    LITERAL_WHILE, LITERAL_DO, LITERAL_ELSE, LITERAL_FINALLY, LITERAL_FOR,
-                    LITERAL_IF, LITERAL_SYNCHRONIZED"/>
+                    INTERFACE_DEF, LAMBDA, METHOD_DEF, OBJBLOCK,
+                    STATIC_INIT, RECORD_DEF, COMPACT_CTOR_DEF"/>
     </module>
     <module name="LeftCurly">
       <property name="id" value="LeftCurlyNl"/>
       <property name="option" value="nl"/>
       <property name="tokens"
-                value="LITERAL_CASE, LITERAL_DEFAULT" />
+                value="LITERAL_CASE, LITERAL_DEFAULT, LITERAL_IF, LITERAL_ELSE, LITERAL_SWITCH,
+                       LITERAL_SYNCHRONIZED, LITERAL_FOR, LITERAL_WHILE, LITERAL_DO,
+                       LITERAL_CATCH, LITERAL_TRY, LITERAL_FINALLY" />
     </module>
```

Comment:
Umm, this is not we want. I guess you're getting wrong idea about the feature we want. 

Go through this stackoverflow post: https://stackoverflow.com/questions/1563030/anonymous-code-blocks-in-java

We want such anonymous code block's opening curly brace ( `{` ) to be on new line. Example from the post:

```java
public static void main(String[] args) {
    // in
    {
        // out
    }
}
```

Here, we have anonymous inner block inside the main method. We want to ensure that such block's opening brace is preceded with a line break.

```java
public static void main(String[] args) {
    // in
    final String str = "s"; { // warn, incorrect placement of "{"
        // out
    }
    final int xyz = 19;
    { // ok, correct placement. "{" is preceded with a line break
       // code here
    }
}
```

In the above example, for `String str` line, we should give warning like `"{" should be alone on the line`.

Here's a small explanation about the from google side: https://github.com/google/styleguide/commit/49488412b2f59843fce2433bd834a3fd9700604e#r143141977, we have some examples in the issue's description too: https://github.com/checkstyle/checkstyle/issues/15324.

With the current changes you made, we're enforcing the opening curly braces of various blocks like if/else, try/catch/finally to be on new line but the rule isn't about that.